### PR TITLE
[ET-912] Ensure the Attendees List title for the Attendees report is clean and not replicated on the page for the admin area as well as frontend

### PR DESCRIPTION
**Pull request by cliffordp**
[ET-912]

There are two different areas the title appears from:

## Posts with RSVPs only (Attendees report)

image

## Posts with at least one ticket (Order tabbed view, that shows Attendees report)

image

[ET-912]: https://moderntribe.atlassian.net/browse/ET-912

### CODE REVIEW

File: src/admin-views/attendees.php [hunk 1]
```diff
@@ -11,31 +11,21 @@
...
  *
  * @since  4.6.2
  * @since  4.12.1 Append the post ID to the Attendees page title and each Ticket's name.
+ * @since  TBD Tweak wording from "Attendees" to "Attendees List" and change default from enabled to disabled.
  *
  * @param boolean                         $show_title
  * @param Tribe__Tickets__Tickets_Handler $handler
  */
-$show_title = apply_filters( 'tribe_tickets_attendees_show_title', true, $attendees );
+$show_title = apply_filters( 'tribe_tickets_attendees_show_title', false, $attendees );
```
Comment (sc0ttkclark) [inline src/admin-views/attendees.php:23]: Would it make sense to use the `is_admin()` as the default?

```suggestion
$show_title = apply_filters( 'tribe_tickets_attendees_show_title', is_admin(), $attendees );
```

**Review by sc0ttkclark — APPROVED** [review]

File: readme.txt [hunk 2]
```diff
@@ -120,6 +120,7 @@ Currently, the following add-ons are available for Event Tickets:
 
 = [5.0.1] TBD =
 
+* Fix - Ensure the Attendees List title for the Attendees report is clean and not replicated on the page for the admin area as well as frontend. [ET-861]
```
Comment (sc0ttkclark) [inline readme.txt:123]:
```suggestion
* Fix - Ensure the Attendees List title for the Attendees report is clean and not replicated on the page for the admin area as well as frontend. [ET-912]
```

**Review by bordoni — DISMISSED** [review]

File: src/admin-views/attendees.php [hunk 3]
```diff
@@ -28,9 +29,9 @@
 			<?php
 			echo esc_html(
 				sprintf(
-				// Translators: 1: the post title, 2: the post ID.
+					// Translators: %1$s: the post title, %2$d: the post ID.
```
Comment (juanfra) [inline src/admin-views/attendees.php:32]: The same comment here about context for translators and maybe including `post/event`

**Review by juanfra — DISMISSED** [review]
Looking good! Just a minor comment

File: src/Tribe/Commerce/Orders_Tabbed_View.php [hunk 4]
```diff
@@ -32,12 +32,26 @@ public function render() {
 		}
 
 		$view_title = sprintf(
-		// Translators: 1: the post title, 2: the post ID.
-			esc_html_x( 'Attendees for: %s [#%d]', 'attendees report screen heading', 'event-tickets' ),
+			// Translators: %1$s: the post title, %2$d: the post ID.
```
[the remark this answers is not in the document]
Comment (sc0ttkclark) [reply src/Tribe/Commerce/Orders_Tabbed_View.php:35]:
```suggestion
			// Translators: %1$s: the post/event title, %2$d: the post/event ID.
```

File: src/admin-views/attendees.php (hunk 3 above)
Comment (sc0ttkclark) [reply src/admin-views/attendees.php:32]:
```suggestion
					// Translators: %1$s: the post/event title, %2$d: the post/event ID.
```

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**